### DISCUSSION TITLE
Vercel Deploy error

### DISCUSSION THREAD

**Discussion by jpeseWang** [vercel/community#2491] [Help] [answered]
### Summary

Hi guys, I'm a newbie to Verecel. Everything was okay since I deploy an old project.

01:11:22.093 | Running build in Cleveland, USA (East) – cle1
-- | --
01:11:22.143 | Cloning github.com/jpeseWang/reactjsPorfolio (Branch: main, Commit: 3121dd1)
01:11:22.377 | Previous build cache not available
01:11:23.384 | Cloning completed: 1.241s
01:11:23.524 | Running "vercel build"
01:11:24.004 | Vercel CLI 29.3.3
01:11:24.543 | Installing dependencies...
01:11:28.373 | npm WARN deprecated w3c-hr-time@1.0.2: Use your platform's native performance.now() and performance.timeOrigin.
01:11:28.685 | npm WARN deprecated stable@0.1.8: Modern JS already guarantees Array#sort() is a stable sort, so this library is deprecated. See the compatibility table on MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#browser_compatibility
01:11:28.764 | npm WARN deprecated rollup-plugin-terser@7.0.2: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-terser
01:11:28.784 | npm WARN deprecated sourcemap-codec@1.4.8: Please use @jridgewell/sourcemap-codec instead
01:11:30.714 | npm WARN deprecated svgo@1.3.2: This SVGO version is no longer supported. Upgrade to v2.x.x.
01:11:40.227 | npm notice
01:11:40.228 | npm notice New minor version of npm available! 9.5.0 -> 9.6.6
01:11:40.228 | npm notice Changelog: <https://github.com/npm/cli/releases/tag/v9.6.6>
01:11:40.228 | npm notice Run `npm install -g npm@9.6.6` to update!
01:11:40.228 | npm notice
01:11:40.230 | npm ERR! code 1
01:11:40.230 | npm ERR! path /vercel/path0/node_modules/core-js
01:11:40.231 | npm ERR! command failed
01:11:40.231 | npm ERR! command sh -c node -e "try{require('./postinstall')}catch(e){}"
01:11:40.231 | npm ERR! node: /lib64/libm.so.6: version `GLIBC_2.27' not found (required by node)
01:11:40.231 | npm ERR! node: /lib64/libc.so.6: version `GLIBC_2.28' not found (required by node)
01:11:40.232 |  
01:11:40.233 | npm ERR! A complete log of this run can be found in:
01:11:40.233 | npm ERR!     /vercel/.npm/_logs/2023-05-17T18_11_24_828Z-debug-0.log
01:11:40.264 | Error: Command "npm install" exited with 1
01:11:40.916 | Deployment completed
01:11:40.522 | BUILD_UTILS_SPAWN_1: Command "npm install" exited with 1

anyone can help me with this error. Tks alots!!

**Comment by smaeda-ks** [comment, marked as answer]
@jpeseWang You really should not include `node` there:
https://github.com/jpeseWang/reactjsPorfolio/blob/main/package.json#L10

it's already available in the build container environment on Vercel, and you don't need to add that at all. Try removing it and update the lock file accordingly, then re-deploy it. Furthermore, the available Node versions are listed here:
https://vercel.com/docs/concepts/functions/serverless-functions/runtimes/node-js#node.js-version
